tests/test_component/Sources/test_component/test_component+Generics.swift

```diff
@@ -7237,7 +7237,7 @@ internal enum __x_ABI_C__FIReference_1___x_ABI_Ctest__zcomponent__CSignedBridge:
 
     static func from(abi: ComPtr<CABI>?) -> SwiftProjection? {
         guard let val = abi else { return nil }
-        var result: Signed = .init(0)
+        var result: __x_ABI_Ctest__component_CSigned = .init(0)
```

Comment:
huh...how did this compile before?

Reply:
oh, it's an enum, so the swift type name is just an alias for the mangled one...interesting